gremlin-dotnet/src/Gremlin.Net/Structure/IO/GraphBinary/TypeSerializerRegistry.cs

```diff
@@ -199,7 +199,7 @@ public ITypeSerializer GetSerializerFor(Type valueType)
                 return serializer;
             }
 
-            foreach (var supportedType in _serializerByType.Keys)
+            foreach (var supportedType in new List<Type>(_serializerByType.Keys))
```

Comment:
What if serializer removed when iteration already started?

Reply:
Removing a serializer isn't really supported so I don't see how that could happen.
Do you see a good use case where someone would actually want to remove a serializer?

Reply:
yes, no deletion